Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -47,19 +47,19 @@ public enum OrientationMode
             None = 0,
 
             /// <summary>
-            /// Face the tracked transform, but always oriented up or down
+            /// Face the tracked transform
             /// </summary>
-            Vertical,
+            TrackedTarget = 1,
 
             /// <summary>
             /// Aligned to surface normal completely
             /// </summary>
-            Full,
+            SurfaceNormal = 2,
```

Comment:
>SurfaceNormal [](start = 12, length = 13)

Would it be hard to add a test case for one of these (just so that we can get some coverage for these options) - maybe the blended case because that could cover most ground?

Reply:
I added TrackedTarget & SurfaceNormal into the original wall test for SurfaceMagnetism

I also changed the lerptime properties so we can test faster for the solvers